lib/kafo/hook_context.rb

```diff
@@ -61,6 +61,13 @@ def add_module(module_name, mapping = nil)
       self.kafo.add_module(module_name)
     end
 
+    # Check if a module is enabled in the current configuration.
+    # examples:
+    #   module_enabled?('example')
+    def module_enabled?(module_name)
+      self.kafo.module(module_name).enabled?
```

Comment:
`Configuration#module_enabled?` isn't authoritative, as the PuppetModule itself contains an enable/disable flag which takes precedence. When a module's disabled from the CLI or wizard, the flag on the PuppetModule is toggled, while the data hash (from the answers file) in `Configuration` remains.

I'm wary of changing how `Configuration#module_enabled?` works (e.g. to check the module status) as `PuppetModule.enabled?` already calls back to this method.

In short: the enabled? method on PuppetModule itself is the main entrypoint.